stacks-signer/src/chainstate.rs

```diff
@@ -566,6 +574,22 @@ impl SortitionsView {
                     "proposed_chain_length" => block.header.chain_length,
                     "expected_at_least" => info.block.header.chain_length + 1,
                 );
+                if info.signed_group.unwrap_or(get_epoch_time_secs())
+                    + reorg_attempts_activity_timeout.as_secs()
+                    > get_epoch_time_secs()
```

Comment:
How about this instead to make the intent more clear:
```suggestion
                if info.signed_group.map_or(true, |signed_time| {
                    signed_time + reorg_attempts_activity_timeout.as_secs() > get_epoch_time_secs()
                }) {
```